src/hitchhiker/konserve.cljc

```diff
@@ -0,0 +1,162 @@
+(ns hitchhiker.konserve
+  (:refer-clojure :exclude [resolve subvec])
+  (:require [clojure.core.rrb-vector :refer [catvec subvec]]
+            #?(:clj [clojure.core.async :refer [chan promise-chan put!] :as async]
+               :cljs [cljs.core.async :refer [chan promise-chan put!] :as async])
+            [konserve.core :as k]
+            [konserve.memory :refer [new-mem-store]]
+            [hasch.core :refer [uuid]]
+            [clojure.set :as set]
+            #?(:clj [hitchhiker.tree.core :refer [go-try <?] :as core]
+               :cljs [hitchhiker.tree.core :as core])
+            [hitchhiker.tree.messaging :as msg])
+  #?(:cljs (:require-macros [hitchhiker.tree.core :refer [go-try <?]])))
+
```

Comment:
Probably that should go into the official konserve documentation. Keys can be arbitrary edn and the values are stored under their first key as granularity: `(k/assoc-in store [:foo :bar] 42)` will create a blob for `:foo` with `{:bar 42}` as value currently. But this is not determined by the protocol itself, backends could handle it differently. I have to rethink the read-handlers atm. because they have some limitations in the way I designed them, I will ping back once I have figured that out.